sound/soc/sof/sof-pci-dev.c

```diff
@@ -233,7 +246,6 @@ static int sof_pci_probe(struct pci_dev *pci,
 	 * to add a dedicated ops field in the generic soc-acpi structure
 	 * to avoid such issues
 	 */
-	mach->pdata = (void *)ops;
 
 	sof_pdata->id = pci_id->device;
```

Comment:
can you remind me what we do with this id?

Reply:
@keyonjie @lgirdwood Any objections to removing this 'id' field? This looks like old stuff that isn't needed.

Reply:
@plbossart the code has updated. Maybe we can merge this PR first if there is no issue on this PR. And make another PR for removing sof_pdata->id if there is on objections.